src/app/(main)/project/[project_id]/page.tsx

```diff
@@ -108,12 +118,18 @@ export default function Chat(params: { params: { project_id: string } }) {
     newChatMutation.mutate({ formdata: fd });
   };
 
-  const samplePrompts = [
-    "How does one prepare for advanced airway management, even before the patient has arrived?",
-    "How do you assess a ICU patient's need for tracheal intubation?",
-    "What are the checks one needs to perform before intubation?",
-    "How do you do the initial assessment of the patient before oxygenation or ventilation?",
-  ];
+  const samplePrompts = useMemo(
+    () =>
+      project?.preset_questions && project?.preset_questions?.length > 0
+        ? project?.preset_questions?.sort(() => Math.random() - 0.5).slice(0, 4)
```

Comment:
Why are you sorting the questions here? Can't the server respond with random questions in each request;

Reply:
That is possible, though I do not see the need to offload this extra logic to backend, the random pick n choose is fine to do on the client side, right? Unless the full prompt list is supposed to be private (which doesn't seem to be the case)..

Reply:
Say I have 100 questions, why should the FE load 100 when its just rendering 4; 🤔

Reply:
Makes sense, I didn't consider that case. 😅